Modules/Core/Common/test/itkImageRangeGTest.cxx

```diff
@@ -148,6 +149,35 @@ namespace
     EXPECT_FALSE(ImageRange<TImage>{ *image }.empty());
   }
 
+
+  template <typename TImage>
+  void ExpectMakeImageRangeReturnsEmptyRangeForNullptr()
+  {
+    TImage* const imageNullptr = nullptr;
+    EXPECT_TRUE(MakeImageRange(imageNullptr).empty());
```

Comment:
Why not also test non-const? `TImage* imageNullptr2 = nullptr;`

Reply:
There is no fundamental difference between passing a const pointer or a non-const pointer (`imageNullptr2`) to this specific function. Because the `argument` of `MakeImageRange` is just a pointer, not a reference-to-a-pointer. (So it cannot modify the pointer.)

I just added this `const` as I'm trying to follow the classical C++ guideline, "Use const whenever possible". Some tools might actually check if the code is according to this rule, and otherwise produce a warning, or note (PC-lint does produce something like: "note: variable could be const").

OK?